test/scripts/hello_world_on_chain.txtar

```diff
@@ -0,0 +1,5 @@
+# [execute]
+exec npx hardhat test --grep $TEST_CASE
```

Comment:
I would suggest using `make test-local` to try to more closely follow what is in the writeup (and get a little coverage on the `Makefile`).

Reply:
updated to use `make test-local`